### PR TITLE
Update syntax to Python 3

### PR DESCRIPTION
**Pull request by hramezani**

### CODE REVIEW

**Comment by diazona** [comment]
Thanks, this makes sense. Let's give it a while and see what the other maintainers say, if they see this anytime soon.

The addition of the badges seems out of place in this pull request... personally I would have split that to a separate PR, but it's not a big deal.

File: README.rst [hunk 1]
```diff
@@ -1,3 +1,11 @@
+.. image:: https://img.shields.io/pypi/v/pytest-localserver.svg?style=flat
+    :alt: PyPI Version
+    :target: https://pypi.python.org/pypi/pytest-django
+
+.. image:: https://img.shields.io/pypi/pyversions/pytest-localserver.svg
+    :alt: Supported Python versions
+    :target: https://pypi.python.org/pypi/pytest-django
```
Comment (diazona) [inline README.rst:7]: Django? :wink: I'm guessing these URLs are meant to point at pytest-localserver instead.

Comment (hramezani) [reply README.rst]: Ah, you are right. It was a copy from `pytest-django` :smile: 
I've fixed it!

**Comment by hramezani** [comment]
> The addition of the badges seems out of place in this pull request... personally I would have split that to a separate PR, but it's not a big deal.

As I keep it in a separate commit, I think you can merge the PR with a `Rebase and Merge`. Then we will have two separate commits.

**Comment by diazona** [comment]
> As I keep it in a separate commit, I think you can merge the PR with a `Rebase and Merge`. Then we will have two separate commits.

Well, that wouldn't be consistent with the merge workflow we've been using. (Merge commits help keep track of features on master and keep the development history well organized.)

**Comment by hramezani** [comment]
@diazona I've removed the badges commit from this PR and created #38

**Review by RonnyPfannschmidt — COMMENTED** [review]
I recommended trying shed/pyupgrade

**Pull request merged**